File: parte001/ex010_triangulo_equilatero.cs
// Dado  el  valor  del  lado  en  un  triángulo  equilátero, haga  un  algoritmo  que  obtenga  el 
// perímetro, el valor de la altura y el área del triángulo.

using System;

namespace Parte001
{
    class Program
    {
        static void Main(string[] args)
        {
            double lado, perimetro, altura, area;

            Console.Write("Ingrese el valor del lado del triángulo equilátero: ");
            lado = Convert.ToDouble(Console.ReadLine());

            perimetro = lado * 3;
            altura = Math.Sqrt(3) * lado / 2;
            area = Math.Pow(lado, 2) * Math.Sqrt(3) / 4;

            Console.WriteLine("El perímetro del triángulo equilátero es: " + perimetro);
            Console.WriteLine("La altura del triángulo equilátero es: " + altura);
            Console.WriteLine("El área del triángulo equilátero es: " + area);

            Console.ReadKey();
        }
    }
}

File: parte001/ex010_triangulo_equilatero.js
// Dado  el  valor  del  lado  en  un  triángulo  equilátero, haga  un  algoritmo  que  obtenga  el 
// perímetro, el valor de la altura y el área del triángulo.

const lado = 5;

const perimetro = 3 * lado;
const altura = Math.sqrt(3) * lado / 2;
const area = (lado * altura) / 2;

console.log("El perímetro del triángulo es: " + perimetro);
console.log("La altura del triángulo es: " + altura);
console.log("El área del triángulo es: " + area);
